main.py

```diff
@@ -0,0 +1,25 @@
+import os
+import urllib.request
+
+import requests
+from dotenv import load_dotenv
+
+if __name__ == '__main__':
+
+    topic = 'cat'
+
+    load_dotenv()
+    API_ACCESS_KEY = os.getenv('API_ACCESS_KEY')
+
+    response = requests.get(f'https://api.unsplash.com/search/photos?client_id={API_ACCESS_KEY}&page=1&query={topic}')
+
+    data = {}
```

Comment:
Pythonists usually don't init an empty default object if there's not a good reason. I saw this solution in some people's code but I suppose they switched to Python from cpp of java. For now let's remove `data = {}` and add some code to process situations when status_code != 200 or parsing response to json failed.